### PR TITLE
Course Provisioning: Add expandable help text for selection of rosters

### PR DESCRIPTION
**Pull request by redconfetti**
https://jira.ets.berkeley.edu/jira/browse/CLC-4711

### CODE REVIEW

File: app/assets/stylesheets/_canvas_course_provision.scss [hunk 1]
```diff
@@ -92,6 +92,18 @@
     background: linear-gradient($bc-color-form-button-gradient1, $bc-color-form-button-gradient2);
   }
 
+  .cc-page-course-provision-help-notice {
+    margin-bottom: 20px;
+    ol {
```
Comment (christianv) [inline app/assets/stylesheets/_canvas_course_provision.scss]: @redconfetti can we use classes instead?

File: app/assets/stylesheets/_canvas_course_add_user.scss [hunk 2]
```diff
@@ -22,23 +22,17 @@
...
-
-  .bc-user-search-notice-container {
-    margin: 4px 1px;
-    min-height: 15px;
-    .bc-user-search-notice-button {
-      color: $bc-color-help-link-blue;
-      font-size: 12px;
+  .bc-user-search-notice {
+    font-size: 13px;
+    dl {
```
Comment (christianv) [inline app/assets/stylesheets/_canvas_course_add_user.scss]: @redconfetti can we use classes instead?

File: app/assets/templates/canvas_embedded/course_provision.html [hunk 3]
```diff
@@ -92,6 +92,26 @@
...
               <div class="cc-page-provision-step-indent">
                 <p>All official sections you select below will be put in ONE, single course site.</p>
+
+                <div class="cc-page-help-notice cc-page-course-provision-help-notice">
+                  <i class="fa fa-question-circle cc-left cc-page-help-notice-icon"></i>
+                  <div class="cc-page-help-notice-left-margin">
+                    <button class="cc-button-link cc-page-help-notice-toggle-button" data-ng-click="toggle.displayHelp = !toggle.displayHelp">
+                      Need help deciding which rosters to select?
+                    </button>
+                    <div class="cc-page-help-notice-content" data-ng-show="toggle.displayHelp">
```
Comment (christianv) [inline app/assets/templates/canvas_embedded/course_provision.html]: @redconfetti does `ng-if` work here?

File: app/assets/templates/canvas_embedded/course_provision.html [hunk 4]
```diff
@@ -92,6 +92,26 @@
...
+                      <p>If you have a course with multiple sections, you will need to decide whether you want to:</p>
+                      <ol>
+                        <li>Create one, single course site which includes official sections for both your primary and secondary sections, or</li>
```
Comment (christianv) [inline app/assets/templates/canvas_embedded/course_provision.html]: @redconfetti seems weird to have a `, or` at the end of the sentence, is that intentional?

File: app/assets/templates/canvas_embedded/course_provision.html [hunk 5]
```diff
@@ -92,6 +92,26 @@
...
+                        <li>Create multiple course sites, perhaps with one for each section.</li>
+                      </ol>
+                      <p>Read about the pros and cons of each choice in thie Help document:
```
Comment (christianv) [inline app/assets/templates/canvas_embedded/course_provision.html]: @redconfetti `thie` => `this`? Also, should `Help` be `help`?

File: app/assets/templates/canvas_embedded/course_provision.html (hunk 4 above)
Comment (redconfetti) [inline app/assets/templates/canvas_embedded/course_provision.html]: Yes. It's intentional. See [the design](https://confluence.ets.berkeley.edu/confluence/download/attachments/38078750/ver6_Canvas%202.png?version=5&modificationDate=1399995187000&api=v2).

File: app/assets/stylesheets/_canvas_embedded.scss [hunk 6]
```diff
@@ -336,4 +339,32 @@ $bc-header-font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif;
     }
   }
 
+  // Collapsed help notice area
+  .cc-page-help-notice {
+    margin: 4px 1px;
+    min-height: 15px;
```
Comment (christianv) [inline app/assets/stylesheets/_canvas_embedded.scss]: @redconfetti interesting, why did we need this?

File: app/assets/templates/canvas_embedded/course_provision.html (hunk 4 above)
Comment (christianv) [inline app/assets/templates/canvas_embedded/course_provision.html]: @redconfetti can you ask the person that designed it whether it's ok to remove it?

File: app/assets/stylesheets/_canvas_embedded.scss (hunk 6 above)
Comment (redconfetti) [inline app/assets/stylesheets/_canvas_embedded.scss]: I'm trying to establish common help notice styling. The content inside of the collapsible area is unique to the page/application it's displayed in, but the help notice itself is consistent and shared across tools (in this case Find a Person to Add and Course Provisioning tools).

**Comment by redconfetti** [comment]
@christianv Don't merge this just yet. It looks like Jon Hays might have an update to a link I need to add.

**Comment by redconfetti** [comment]
@christianv ok, I've fixed the link. Please let me know if there is anything outstanding that needs attention. Otherwise, please merge :kissing_closed_eyes:

**Comment by raydavis** [comment]
I'm gonna guess that this is ready to go

File: app/assets/stylesheets/_canvas_embedded.scss (hunk 6 above)
Comment (christianv) [inline app/assets/stylesheets/_canvas_embedded.scss]: @redconfetti thanks!

**Pull request merged**